.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+
+## Related issues
+
+* none
+
+## Related code and/or documentation pull requests
+
+* none
+
+## Documentation
+
+- [ ] User facing documentation not needed
+- [ ] Release notes entry not needed
+
+### Release notes section
+
+- [ ] General
+- [ ] Security
+- [ ] Connector(s)
+- [ ] SPI
```

Comment:
how does it help?

Reply:
It forces the contributor and merging maintainer to think through release notes entry from the start and latest at merge. And it creates a standard place with the PR to have that info ..

Reply:
this looks as written by a release notes maniac, as release notes are being now mentioned in 3 different places.
the redundancy is going to irritate, not help

Reply:
Updated as discussed with @electrum